tests/src/main/scala/sttp/tapir/tests/package.scala

```diff
@@ -309,6 +315,41 @@ package object tests {
       .out(stringBody)
       .name("Query with default")
 
+  object MultipleMediaTypes {
+    implicit val schemaForPerson: Schema[Person] = Schema.derived[Person]
+    implicit val schemaForOrganization: Schema[Organization] = Schema.derived[Organization]
+
+    implicit val xmlCodecForOrganization: XmlCodec[Organization] =
+      Codec.xml(xml =>
+        DecodeResult.Value(
+          // <name>xxx</name>
+          Organization(xml.split(">")(1).split("<").head)
+        )
+      )(_ => "sml")
```

Comment:
to reuse the codec we should produce valid xml here, wrapping whatever is incoming into `<name>`, no?